CMake/HPHPFunctions.cmake

```diff
@@ -1,3 +1,5 @@
+cmake_policy(SET CMP0026 OLD)
```

Comment:
The problem is that this call to `cmake_policy` is an error on versions of CMake without that policy -_-...

That message is a warning, not an error.

Once we require CMake >= 3.0 we can get rid of it via generator expressions, but we've not tried to do that yet.